e2e/smoke_test.go

```diff
@@ -78,7 +78,8 @@ var _ = Describe("Smoke End To End Tests - against OpenShift Cluster with Istio
 			<-testshell.Execute(NewProjectCmd(namespace)).Done()
 
 			UpdateSecurityConstraintsFor(namespace)
-			DeployLocalOperator(namespace)
+			PushOperatorImage(namespace)
+			InstallLocalOperator(namespace)
```

Comment:
Split into two steps as I need to push the image for the `install-operator` tests and only then invoke it in different flavours.